docs/meetings/newsletters/2024-01-04_volunteer_newsletter.md

```diff
@@ -0,0 +1,116 @@
+* **Add number of comments to events** [Add number of comments to event tiles](https://github.com/Couchers-org/couchers/issues/5217)
+* **Show user community events on dashboard** [Add events from city communities user belongs to on dashboard](https://github.com/Couchers-org/couchers/issues/5273)
+* **Upgrade bug reporting to include screenshot/video** [Upgrade bug reporting with sentry to include more context, photo, etc](https://github.com/Couchers-org/couchers/issues/5291)
+* **Frontend Package Upgrades**: Help us stay current by upgrading to [React v19](https://github.com/Couchers-org/couchers/issues/5381), [Next.js v14](https://github.com/Couchers-org/couchers/issues/5451), [Material-UI v6](https://github.com/Couchers-org/couchers/issues/5376), and [react-query v3](https://github.com/Couchers-org/couchers/issues/5263), among others.
+* 📱 **React Native Mobile App**: We could use more hands here. Help [Ivan](https://github.com/polemius) tackle the standalone app! We don't have a specific ticket for this as we're making it from scratch, see [how to get involved](#-how-to-get-involved) to help.
+
+Check out our [Couchers Engineering Task Board](https://github.com/Couchers-org/couchers/projects) for more and see [How To Get Involved](#-how-to-get-involved).
+
+---
+
+## 🎨 [Design & Operations Accomplishments](#-design-and-operations-accomplishments)
+
+* 📧 **Graphic Email Notifications**: We’ve moved from plain text emails to a polished, graphic design for notifications.
+* 📝 **Private Feedback Flow**: We’ve designed improvements for private feedback during the reference process (implementation pending). 
+
+Shoutout to [Jesse](https://couchers.org/user/jesse), who’s been doing design and support solo! Any Figma whizzes among you? Come help him!
+
+* ❓ **Couchers 2024 User Survey**: We conducted a survey among all Couchers.org users to gain insight into the main issues and priorities of our users. Improvements to the map search and filtering of inactive/incomplete profiles were the most requested feautures in the survey. Shoutout to [Yannic](https://github.com/spreeni) who had the main role in this!
+
+### ✍️ [Help Wanted: Design](#️-help-wanted-design)
+
+1. 🗺️ **Map Redesign**: Our top focus. We've talked about it a lot, but we need someone with design thinking to help us make a decision! Want to help?
+2. 🛡️ **Strong Verification Design**: We could use help with the design for this feature to move it out of Feature Preview. 
+3. 🚀 **Signup Flow & Landing Page**: Needs a major overhaul for better user experience.  
+4. 📱 **Native App Design**: We need designs for the standalone mobile app.
+
+---
+
+## 🛠️ [Help Wanted: Operations](#️-help-wanted-operations)
+
+We could use a hand with:
+
+* 🌍 **Translations**: We’re behind on translating the site into other languages. If you’re interested in managing the translation process or helping with translations, let us know!  
```

Comment:
divide into 

* Translation lead organizer: with the new feature updates and changes to the platform, we've fallen behind on translating the app. If you're interested in helping organize this effort to translate the new parts, please get in touch!
* Translators: just want to help us translate the app into your language? Please get in touch!

I'd be good if we had a shortlist of key languages.

Also add

* Support people